### PR TITLE
sortPoints unit tests done

### PR DESCRIPTION
**Pull request by sourcewire**

### CODE REVIEW

File: src/lib/sortPoints.test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+test('sortPoints correctly sorts list of (x,y) ints', function() {
+        //arrange
+        const array = [{x:12, y:3}, {x:4, y:7}, {x:6, y:12}];//input array
+        const expectedResult = [{x:4, y:7},{x:6, y:12},{x:12, y:3}];//what the array should be after sorting
+
+        //act
+        const result = sortPoints(array);//call function with array param
+
+        //assert
+        expect(result).toStrictEqual(expectedResult)//expect the result to be equal to the expected result
```
Comment (baumannevan) [inline src/lib/sortPoints.test.js:14]: what is stricktequal??

**Review by baumannevan — APPROVED** [review]
Overall looks good. It appears that you have set up test that verify all of the requirements for the sortPoints.js.

File: src/lib/sortPoints.test.js (hunk 1 above)
Comment (sourcewire) [reply src/lib/sortPoints.test.js:14]: i was using .toBe and it gave me an error and suggested i use this haha. I think it just means its a strict match. so the string, result == expected_result

**Review by GrantAOConnor — APPROVED** [review]
Looks good, but maybe the tests should have different descriptions.

**Pull request merged**